### ISSUE TITLE
`./gradlew desktop:run` fails in Linux, "Could not find tools.jar"

### ISSUE THREAD

**Issue by mildmojo** [ChainGangChase/cgc-game#47]
I cloned the repo and ran `./gradlew npm_install` successfully, but `./gradlew desktop:run` fails for me in Linux with OpenJDK:

```
Executing task ':core:compileJava' (up-to-date check took 0.036 secs) due to:
  No history is available.
All input files are considered out-of-date for incremental task ':core:compileJava'.
Compiling with JDK Java compiler API.
:core:compileJava FAILED
:core:compileJava (Thread[Daemon worker Thread 3,5,main]) completed. Took 0.053 secs.

FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':core:compileJava'.
> Could not find tools.jar

* Try:
Run with --stacktrace option to get the stack trace. Run with --debug option to get more log output.

BUILD FAILED

Total time: 2.137 secs
Stopped 0 compiler daemon(s).
Received result Failure[value=org.gradle.initialization.ReportedException: org.gradle.internal.exceptions.LocationAwareException: Execution failed for task ':core:compileJava'.] from daemon DaemonInfo{pid=5747, address=[27ebbade-2602-4532-b618-2969605e2144 port:57386, addresses:[/0:0:0:0:0:0:0:1%1, /127.0.0.1]], idle=true, context=DefaultDaemonContext[uid=0c5b5f09-b38e-4877-87d2-2a2e4b6203bd,javaHome=/usr/lib/jvm/java-7-openjdk-amd64,daemonRegistryDir=/home/shrapnel/.gradle/daemon,pid=5747,idleTimeout=10800000,daemonOpts=-Xms128m,-Xmx1500m,-Dfile.encoding=UTF-8,-Duser.country=US,-Duser.language=en,-Duser.variant]} (build should be done).

myhost ~/Downloads/cgc-game $ java -version
java version "1.7.0_101"
OpenJDK Runtime Environment (IcedTea 2.6.6) (7u101-2.6.6-0ubuntu0.14.04.1)
OpenJDK 64-Bit Server VM (build 24.95-b01, mixed mode)
```

**Comment by Percipient24** [comment]
I haven't tried it with the Open JDK yet. This will be an adventure! :-)

**Comment by Percipient24** [comment]
Alright, I'm running Ubuntu 14.04, but was using `java-8-oracle 1072 /usr/lib/jvm/java-8-oracle` prior to looking at this.

Since 14.04 didn't have openjdk in its default package repos, I needed to run `sudo add-apt-repository ppa:openjdk-r/ppa` to add it, followed by a `sudo apt-get install openjdk-8-jdk` (note that -jdk will also install -jre).

Once openjdk is installed, use `update-alternatives` to switch versions. I didn't know about `update-alternatives` until researching this issue, so thanks for making me learn something new and cool!

`update-java-alternatives -l` will list out which versions of java you have available on your machine.

`sudo update-alternatives --config java` will allow you to interactively select which version of java to point to. Hit the selection number that matches the recently installed openjdk. Also run `sudo update-alternatives --config javac` and select the matching version for `javac`.

After that, `java -version` will report something like:
`openjdk version "1.8.0_91"
OpenJDK Runtime Environment (build 1.8.0_91-8u91-b14-0ubuntu4~14.04-b14)
OpenJDK 64-Bit Server VM (build 25.91-b14, mixed mode)`

I tried a build, and it worked for me (even after a restart, just to be safe). However, I can't be sure it's because I successfully switched to the OpenJDK, or because I have some ghost of the Oracle Java still haunting my system. @mildmojo - give it a go and let me know if any of this helps!
